src/logic/tokens/store/actions/activateAssetsByBalance.js

```diff
@@ -20,6 +20,11 @@ const activateAssetsByBalance = (safeAddress: string) => async (
     await dispatch(fetchCollectibles())
```

Comment:
Should this dispatch be moved below `if (safes.size === 0)` check?

Reply:
indeed! change did https://github.com/gnosis/safe-react/pull/738/commits/4f084dbb6ab73dafc8b83c675f6c5263b820013e